integration/src/test/java/org/kie/hacep/RemoteKieSessionImplTest.java

```diff
@@ -68,6 +68,7 @@ public void getFactCountTest() {
                                                     RemoteKieSession.class);
         try (RemoteKieSessionImpl client = new RemoteKieSessionImpl(Config.getProducerConfig("getFactCountTest"),
                                                                     topicsConfig)) {
+            client.fireUntilHalt();
             client.listen();
```

Comment:
This is not part of this PR but can you please verify what is `listen()`?
I do not expect user need to explicitly start the client. In any case, if we keep this command I think it needs to be the first method to be invoked